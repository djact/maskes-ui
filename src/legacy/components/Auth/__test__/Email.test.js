import React from 'react'
import { shallow } from 'enzyme'
import { EmailReset, EmailResetConfirm } from '..'

describe('Test Email Reset Components', () => {
    it('should shallow render correctly EmailReset', () => {
        const EmailResetComponent = shallow(<EmailReset />)
        expect(EmailResetComponent).toMatchSnapshot()
    })

    it('should shallow render correctly EmailResetConfirm', () => {
        const EmailResetConfirmComponent = shallow(<EmailResetConfirm />)
        expect(EmailResetConfirmComponent).toMatchSnapshot()
    })
})
