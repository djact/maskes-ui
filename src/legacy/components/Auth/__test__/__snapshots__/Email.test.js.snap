// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Email Reset Components should shallow render correctly EmailReset 1`] = `
<Container
  className="email-reset-container"
  fluid={false}
>
  <Form
    className="email-reset-form"
    inline={false}
    onSubmit={[Function]}
  >
    <FormGroup
      controlId="EmailResetEmail"
    >
      <FormLabel
        column={false}
        srOnly={false}
      >
        Send Email Reset Link
      </FormLabel>
      <FormControl
        name="email"
        onChange={[Function]}
        placeholder="Enter Email"
        required={true}
        type="email"
        value=""
      />
    </FormGroup>
    <Button
      active={false}
      disabled={false}
      type="submit"
      variant="primary"
    >
      Submit
    </Button>
  </Form>
</Container>
`;

exports[`Test Email Reset Components should shallow render correctly EmailResetConfirm 1`] = `
<Container
  className="email-reset-container"
  fluid={false}
>
  <Form
    className="email-reset-form"
    inline={false}
    onSubmit={[Function]}
  >
    <FormGroup
      controlId="formBasicResetEmail"
    >
      <FormLabel
        column={false}
        srOnly={false}
      >
        New Email
      </FormLabel>
      <FormControl
        name="new_email"
        onChange={[Function]}
        placeholder="Enter New Email"
        required={true}
        type="email"
      />
    </FormGroup>
    <FormGroup
      controlId="formBasicRetypeEmail"
    >
      <FormLabel
        column={false}
        srOnly={false}
      >
        Retype Email
      </FormLabel>
      <FormControl
        name="re_new_email"
        onChange={[Function]}
        placeholder="Retype New Email"
        required={true}
        type="email"
      />
    </FormGroup>
    <Button
      active={false}
      disabled={false}
      type="submit"
      variant="primary"
    >
      Submit
    </Button>
  </Form>
</Container>
`;
